config/initializers/lti_message_store.rb

```diff
@@ -2,11 +2,16 @@
 
 module GitHubClassroom
   def self.lti_message_store(options = {})
+    if options[:lti_configuration].present?
+      options[:consumer_key] = options[:lti_configuration].consumer_key
+      options.delete(:lti_configuration)
+    end
```

Comment:
Does this need a test?

Reply:
Great question! Meant to highlight this when I submitted the PR. I didn't see any tests for initializers within the specs, is there a specific place I should create a test for this?

Reply:
You should create a test at `spec/config/initializers/lti_message_store_spec.rb`

Reply:
Since this is an initializer I suppose it doesn't need a test 🤷‍♂